app/Models/FedCap.php

```diff
@@ -53,11 +53,16 @@ public function getRemainingCapAttribute() {
             $total += $cap->total_attestations;
         }
 
-        if ($total < 0) {
-            return $this->total_attestations + $total;
+        return $this->total_attestations - $total;
+    }
+
+    public function getRemainingReservedGraduateCapAttribute() {
+        $total = 0;
```

Comment:
Can you update this and in the other function to avoid using a loop.
$total = $this->caps->sum('total_reserved_graduate_attestations');